plugin-flex-ts-template-v2/src/feature-library/canned-responses/custom-components/CannedResponsesCRM/Response/Response.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Actions, ITask, useFlexSelector } from '@twilio/flex-ui';
+
+import { Text } from '@twilio-paste/text';
+import { Button } from '@twilio-paste/core/button';
+import { EditIcon } from '@twilio-paste/icons/esm/EditIcon';
+import { SendIcon } from '@twilio-paste/icons/esm/SendIcon';
+import { Tr, Td } from '@twilio-paste/table';
+import { Flex } from '@twilio-paste/flex';
+import { Tooltip } from '@twilio-paste/tooltip';
+
+interface ResponseProps {
+  label: string;
+  text: string;
+  task: ITask;
+}
+
+const Response: React.FunctionComponent<ResponseProps> = ({ text, task }) => {
+  const inputState = useFlexSelector(
+    (state) => state.flex.chat.conversationInput[task.attributes.conversationSid].inputText,
+  );
+
+  const onClickSend = () => {
+    if (!task.attributes.conversationSid) return;
+    Actions.invokeAction('SendMessage', { body: text, conversationSid: task.attributes.conversationSid });
```

Comment:
It looks like this causes any existing input to be lost :( If you await this action, afterwards you can invoke SetInputText with the previous input and that solves the issue. https://github.com/dremin/plugin-chat-commands/blob/main/src/commands/smileCommand.js